pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+{package} {version} {:snowflake:}
+
+**Destination channel:** {Snowflake | defaults}
+
+### Links
+
+- [{ticket_number}]() 
+- [Upstream]()
+- [conda-forge]()
+- [pypi]()
```

Comment:
Are those needed? I think a changelog/diff link here may be more useful.

Reply:
I'd agree that the conda-forge link is redundant, since the vast majority of our feedstocks are forks from conda-forge, and therefore have a link embedded in the repository already. Barring any objections, I'll remove it.

I'm not clear on the benefit of a pypi link. Could you please shed some light on this @lorepirri ? 

A changelog/diff link would be a great addition.